src/schema/types/mutations/Project.ts

```diff
@@ -29,10 +41,9 @@ export const deleteProject = {
   args: {
     id: nonNull(intArg()),
   },
-  resolve(_: any, args: any, ctx: any) {
-    // 후에 프로젝트 삭제 구현
-    const mock_error = "mock error!";
-    return mock_error;
+  resolve: async (_: any, args: any, ctx: any) => {
```

Comment:
여기도 `ctx`가 사용이 안되니, `__`처리 하는것이 좋다고 생각되네요
```suggestion
  resolve: async (_: any, args: any, __: any) => {
```